2020-12-29 18:17:51,272 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2020-12-29 18:17:51,272 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Users/oak/opt/anaconda3/lib/python3.7/site-packages/instabot/bot/bot.py
2020-12-29 18:17:51,274 - instabot version: 0.117.0 (api_login) - DEBUG - Loading cookies
2020-12-29 18:17:51,275 - instabot version: 0.117.0 (api_login) - DEBUG - Loading uuids
2020-12-29 18:17:51,275 - instabot version: 0.117.0 (api_login) - INFO - Recovery from /Users/oak/Work/InstaBot/tg_bot/config/oaks_bot_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=1af75152-18c2-4e49-aacf-9c4d86c5b00a
- uuid=acb79310-00a5-46c3-aa26-4c539f318181
- client_session_id=72bdd548-5011-4984-a7e2-d51ea6d7fda3
- device_id=android-4521b2fc925cd29a
2020-12-29 18:17:51,275 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-12-29 18:17:52,587 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [200]>
2020-12-29 18:17:53,154 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [200]>
2020-12-29 18:17:53,783 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: direct_v2/ranked_recipients/ returned response: <Response [200]>
2020-12-29 18:17:54,180 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: direct_v2/ranked_recipients/ returned response: <Response [200]>
2020-12-29 18:17:54,695 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: direct_v2/inbox/ returned response: <Response [200]>
2020-12-29 18:17:55,301 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: direct_v2/get_presence/ returned response: <Response [200]>
2020-12-29 18:17:55,997 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: news/inbox/?limited_activity=true&show_su=true returned response: <Response [200]>
2020-12-29 18:17:56,355 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: users/profile_notice/ returned response: <Response [200]>
2020-12-29 18:17:57,658 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: discover/explore/ returned response: <Response [200]>
2020-12-29 18:17:57,659 - instabot version: 0.117.0 (api) - INFO - Logged-in successfully as 'oaks_bot'!
2020-12-29 18:17:57,659 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Loading Checkpoint file from: /Users/oak/Work/InstaBot/tg_bot/config/oaks_bot.checkpoint
2020-12-29 18:18:04,667 - instabot version: 0.117.0 (bot_get) - ERROR - Unexpected link
2020-12-29 18:18:05,236 - instabot version: 0.117.0 (api) - DEBUG - GET to endpoint: media/False/info/ returned response: <Response [400]>
2020-12-29 18:18:05,237 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "Invalid media_id False", "status": "fail"}'
2020-12-29 18:18:05,237 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-12-29 18:18:05,237 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: Invalid media_id False
2020-12-29 18:18:05,237 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2020-12-29 18:18:05,237 - instabot version: 0.117.0 (bot_get) - INFO - Media with False not found.
2020-12-29 18:19:25,311 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: /Users/oak/Work/InstaBot/tg_bot/config/oaks_bot.checkpoint
2020-12-29 18:19:25,312 - instabot version: 0.117.0 (bot) - INFO - Total requests: 266
